mls-rs-core/src/time.rs

```diff
@@ -57,6 +57,15 @@ extern "C" {
     fn date_now() -> f64;
 }
 
+#[cfg(all(target_arch = "wasm32", feature = "node"))]
+#[wasm_bindgen(inline_js = r#"
+module.exports.date_now = function() {
+    return Date.now();
+};"#)]
+extern "C" {
+    fn date_now() -> f64;
+}
+
```

Comment:
Does this mean that I can't compile this for `wasm32` with `--no-default-features` (since date_now is undefined) or `--all-features` (since date_now is defined twice)? I think in rust features should be additive. Maybe something like `not(feature = "web")` here? Not sure

Reply:
oh ok I see, I will try to make them additive, thanks!

Reply:
I think I have implemented a good solution now